src/store.js

```diff
@@ -0,0 +1,28 @@
+// src/store.js
+import { configureStore, createSlice } from '@reduxjs/toolkit';
+
+const initialState = {
+  selectedAuthor: '',
```

Comment:
add posts here and store all the posts in store as well with localstorage

Reply:
done